llm_studio/src/plots/text_causal_language_modeling_plots.py

```diff
@@ -176,7 +181,12 @@ def plot_validation_predictions(
             # exclude last answer
             answers[-1] = ""
             for prompt, answer in zip(prompts, answers):
-                input_text += prompt + answer
+                input_text += (
+                    f" **{prompt_column_name}:** "
+                    f"{prompt} "
+                    f"**{cfg.dataset.answer_column}:** "
```

Comment:
let's add a newline or two here for better readability

Reply:
I've added newlines after each prompt/answer.
![Screenshot 2023-10-10 at 10-14-55 Screenshot from 2023-10-10 10-13-45 png (PNG Image 5120 × 2880 pixels) — Scaled (46%)](https://github.com/h2oai/h2o-llmstudio/assets/24281881/0e0e1748-443a-4e36-a7a3-faed17124763)